Source/gg2/Objects/InGameElements/Character.events/User Event 13.xml

```diff
@@ -54,12 +54,14 @@ if(global.updateType == FULL_UPDATE){
             break;
         case CLASS_SNIPER:
             currentWeapon.t = read_ubyte(global.deserializeBuffer);
+            if(currentWeapon.t) zoomed = true; // Quick hack
```

Comment:
It's a hack because, technically, you can be zoomed with 0 t. I think. Not sure how the code execution order works.